app/src/main/res/values-night/colors.xml

```diff
@@ -93,7 +93,7 @@
     <color name="collection_icon_color_yellow">@color/collection_icon_color_yellow_dark_theme</color>
 
     <!-- Home screen -->
-    <color name="home_show_all_button_text">@color/photonLightGrey50</color>
+    <color name="home_show_all_button_text">#AB71FF</color>
```

Comment:
This should be equivalent to photonViolet40. You can usually see the color name in figma if you select the CSS.

Reply:
<img width="256" alt="image" src="https://user-images.githubusercontent.com/43795363/134415785-818db675-7ed2-4ef0-a935-74101a3a6341.png">

Reply:
Oui, the color variable for photonViolet40 is what I am trying to say ab71ff
https://searchfox.org/mozilla-mobile/source/android-components/components/ui/colors/src/main/res/values/photon_colors.xml#180

Reply:
We don't have that in our .values folder on Fenix. What's the correct way to add a new photon color here?

Reply:
Should simplify be `@color/photonViolet40`

Reply:
That doesn't work unless it's already defined in the .values. I'll use the create wizard and see if that works

Reply:
We'll have to add it to `mozilla.components.ui.colors` to be able to access it the correct way. I'll open a PR for that and we can keep #AB71FF for now

Reply:
It's working fine for me locally. There's really no magic here. We were just using `@color/photonLightGrey50` before the changed to #AB71FF in the PR. 

```suggestion
    <color name="home_show_all_button_text">@color/photonViolet40</color>
```

Reply:
It also already exists in ui-color https://searchfox.org/mozilla-mobile/source/android-components/components/ui/colors/src/main/res/values/photon_colors.xml#180

Reply:
I am just blocking on this change since it should work. Otherwise, I suspect there might be some local build issue.

Reply:
that's in android components. Try it locally. We build our @color/photonXXX values based on https://github.com/mozilla-mobile/android-components/blob/releases/93.0/components/ui/colors/src/main/res/values/photon_colors.xml and you can see that violet40 isn't there

Reply:
<img width="573" alt="image" src="https://user-images.githubusercontent.com/43795363/134572857-f38bbdde-726e-4aec-a6c9-7b6790392023.png">

Reply:
I'm not adding that in this PR. I'm gonna expose it on AC then update here.

Reply:
It's there in AC. https://github.com/mozilla-mobile/android-components/blob/main/components/ui/colors/src/main/res/values/photon_colors.xml#L180. We aren't marked to the 93 release in main.

Trust or ping me for troubleshooting your local dev.

Reply:
We can also omit the change and I can add it in a separate PR if we are still having issues, but the color definitely exists in AC already.